docs/react/guides/dependent-queries.md

```diff
@@ -51,3 +53,33 @@ Once we have the projects, it will go to:
 status: 'success'
 fetchStatus: 'idle'
 ```
+
+## useQueries dependent Query
+
+Dynamic parallel query - `useQueries` can depend on a previous query also, here's how to achieve this:
+
+[//]: # 'Example'
```

Comment:
we need a different name than `Example` here, and it also needs to have a corresponding, closing "tag".

Reply:
I think the Example is commented out, so it doesn't show in the preview.

As for the closing tag, I'm sorry, I don't understand how you mean, could you explain more?

Reply:
see e.g. here:

https://github.com/TanStack/query/blob/716c5d8b096c624f6643dc9f263fbdd6f0565021/docs/react/guides/paginated-queries.md#L8-L15

Reply:
Okay, I get it now, I hope `Example2` will be okay.

However, I don't understand why the syntax is needed or what it does` [//]: # 'Example' `

Reply:
it's a custom renderer that specifies example blocks in react code that can be switched out with other code in other frameworks like vue or svelte

Reply:
Okay, I've changed it to `Example2`.
Is this resolved now?